### PR TITLE
Add Scolcoin Blockchain mainnet and testnet

### PR DESCRIPTION
**Pull request by scolcoin**
Add eip155-65450 "Scolcoin Mainnet"
Add eip155-6552 "Scolcoin Testnet"

### CODE REVIEW

**Comment by scolcoin** [comment]
I am waiting to be able to make modifications and be able to add the Chain of our project

File: _data/chains/eip155-65450.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://scolcoin.com",
+  "shortName": "scol",
+  "chainId": 65450,
+  "networkId": 65450,
+  "explorers": [
+    {
+    "name": "Scolcoin Mainnet Explorer",
+    "url": "https://explorer.scolcoin.com/",
```
Comment (ligi) [inline _data/chains/eip155-65450.json:18]:
```suggestion
    "url": "https://explorer.scolcoin.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `SCOL` more than once

also: run the prettier

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

also: 

I could not resolve your CID with `ipfs get` in a reasonable time - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.

**Review by ligi — APPROVED** [review]

**Pull request merged**